debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -355,7 +355,7 @@ public void initConnection() throws SQLException, InterruptedException {
 
         try (Statement stmt = pgConnection().createStatement()) {
             String createCommand = String.format(
-                    "CREATE_REPLICATION_SLOT %s %s LOGICAL %s",
+                    "CREATE_REPLICATION_SLOT \"%s\" %s LOGICAL \"%s\"",
```

Comment:
I don't think the plug-in name (third parameter) requires quoting, or does it?

Reply:
Right, this change is not needed.